python/shotgun_fields/shotgun_field_manager.py

```diff
@@ -219,6 +250,29 @@ def create_display_widget(self, sg_entity_type, field_name, entity=None, parent=
 
         return None
 
+    def create_delegate(self, sg_entity_type, field_name, view):
```

Comment:
That should be in the model:
https://doc.qt.io/qt-4.8/model-view-programming.html#making-the-model-editable

Definitely going to verify that when we get to editors for the Shotgun fields.